src/test/java/minkostplan/application/DBcontroller/user/UserTest.java

```diff
@@ -0,0 +1,48 @@
+package minkostplan.application.DBcontroller.user;
+import minkostplan.application.entity.Users;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
+import java.time.LocalDateTime;
+@SpringBootTest
+
+class UserRepositoryTest {
+
+    @Mock
+    private UserRepositoryImpl userRepository;
```

Comment:
Ved mindre du skal bruge generic ting, så ville jeg fortrække vi brugte userrepository istedet for impl
```suggestion
    private UserRepository userRepository;
```